net-analyzer/openvas/files/README.gentoo

```diff
@@ -1,34 +1,55 @@
-To allow openvasmd to work, you need to do the following
-
-# Generate a Server Certificate
-openvas-mkcert
-
-# Generate a Client Certificate
-openvas-mkcert-client -n -i
-
-# get nvt feed
-openvas-nvt-sync (free) or greenbone-nvt-sync (commercial)
-
-# get scap feed
-openvas-scapdata-sync (free) or greenbone-scapdata-sync (commercial)
-
-# get cert feed
-openvas-certdata-sync (free) or greenbone-certdata-sync (commercial)
-
-# Generate initial NVT cache (takes some minutes)
-rc-service openvassd create_cache or openvassd --foreground --only-cache
-
-# Start the OpenVAS Scanner
-rc-service openvassd start or systemctl start openvassd.service
-
-# Initialize the Database (This is going to take some time, perhaps time to get coffee?)
-openvasmd --rebuild --progress -v
-
-# Create an admin user and note down the initial password
-openvasmd --create-user=admin --role=Admin
-
-# Start the OpenVAS Manager
-rc-service openvasmd start or systemctl start openvasmd
-
-# Start the Greenbone Security Assistant
-rc-service gsad start or systemctl start gsad
+----------------------------------------------------------------------------------------------------------------------
```

Comment:
Okay, this was pretty nice, but it wraps badly and becomes messy. You should use the default terminal width to wrap these lines, and maybe make it a bit more easily readable. I think the default was 72 or 80 characters, it's in the devmanual.

Reply:
fixed to 72 characters.